app/controllers/api/v1/rentals_controller.rb

```diff
@@ -0,0 +1,30 @@
+class RentalsController < ActionController::Base
+  def create
+    @rental = Rental.new(rental_params)
+    @station = Station.find(@rental[:station_id])
+    @rental.station = @station
+    @user = User.find(params[:user_id])
+    @rental.user = @user
+    unless @rental.save
+      render_error
+    end
```

Comment:
do we want to do something like this, so it will show the redirect to the show page?
if @rental.save
      render :show, status: :created
    else
      render_error
    end

Reply:
good question. I was thinking maybe we can create a new page in wechat app, for when a user creates a session (like a page with a timer and a battery animation)?